Sources/SPTDataLoader.m

```diff
@@ -62,6 +64,7 @@ - (instancetype)initWithRequestResponseHandlerDelegate:(id<SPTDataLoaderRequestR
         _cancellationTokens = [NSMutableArray new];
         _delegateQueue = dispatch_get_main_queue();
         _requests = [NSMutableArray new];
+        _requestHandlers = [NSMapTable new];
```

Comment:
If using `NSMapTable`, use one of the recommended constructors: https://developer.apple.com/documentation/foundation/nsmaptable?language=objc#1661241